sdks/java/core/src/test/java/org/apache/beam/sdk/util/UnownedOutputStreamTest.java

```diff
@@ -53,4 +57,21 @@ public void testClosingThrows() throws Exception {
     expectedException.expectMessage("Caller does not own the underlying");
     os.close();
   }
+
+  @Test
+  public void testWrite() throws IOException {
+    ByteArrayOutputStream expected = new ByteArrayOutputStream();
+    ByteArrayOutputStream actual = new ByteArrayOutputStream();
+    UnownedOutputStream osActual = new UnownedOutputStream(actual);
+
+    writeToBoth(expected, osActual, "Hello World!".getBytes(StandardCharsets.UTF_8));
+    writeToBoth(expected, osActual, "Welcome!".getBytes(StandardCharsets.UTF_8));
+
+    assertArrayEquals(expected.toByteArray(), actual.toByteArray());
+  }
+
+  private static void writeToBoth(OutputStream a, OutputStream b, byte[] data) throws IOException {
+    a.write(data);
+    b.write(data);
```

Comment:
should not this call the method with the full signature?
Maybe it is best to inline this method in the caller, looks simpler IMO

Reply:
inlined with full signature!

Reply:
Done! Inlined with full signature!